src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -113,4 +135,25 @@ export class ItemChildrenEditComponent implements OnChanges {
   onScoreWeightChange(): void {
     this.childrenChanges.emit(this.data);
   }
+
+  onClick(child: ChildData&ChildDataAdditions): void {
+    if (!this.itemData || child.isLocked || !child.id) {
```

Comment:
It means it will be clickable but not do anything?
`!this.itemData` and `!child.id` is not supposed to happen I assume. 
For `child.isLocked`, it is not really like the children list, you can let the user visit it even if locked. I don't think you should care about "islocked" in this component.